integration_tests/sdk/aqueduct_tests/param_test.py

```diff
@@ -14,7 +15,7 @@
 from pandas._testing import assert_frame_equal
 from PIL import Image
 
-from aqueduct import metric, op
+from aqueduct import Local_Data, metric, op
```

Comment:
I have been thinking about this for a while and might need your opinion. So right now, I have `aqueduct.local_data(...)` which creates a `LocalData` class object. The function is basically just a wrapper plus some argument validations. Should we only have the `LocalData` class and put the argument validation in the pydantic base class. When we are designing this we are using `aqueduct.LocalData()` but I am not too sure if we should do everything in the Class Object.

Reply:
Hmm I think we should make our own LocalData class. It can use pydantic within itself, but I think some of our requirements are not so easily enforced in pydantic. For example: the table format should only be supplied if the type is ABLE.

Reply:
A function is also fine... but I like the idea of a class more actually.

Reply:
Done!